pkg/ccl/backupccl/backup_planning.go

```diff
@@ -792,6 +792,14 @@ func backupPlanHook(
 				initialDetails.Destination.Subdir = latestFileName
 			} else if subdir != "" {
 				initialDetails.Destination.Subdir = "/" + strings.TrimPrefix(subdir, "/")
+				// Deprecation notice for `BACKUP INTO` syntax with an explicit subdir.
+				// Remove this once the syntax is deleted in 22.2.
+				p.BufferClientNotice(ctx,
```

Comment:
Just to make sure I understand correctly - we want to deprecate this in BACKUP, but _not_ RESTORE, right? So they can restore from arbitrary points in a series of incremental layers?

Reply:
that's correct! Users of SHOW BACKUP and RESTORE should still be able to pick the specific the full backup in a collection.